react/productSummary.css

```diff
@@ -17,6 +17,10 @@
   width: 281px;
 }
 
+.containerParallel {
+  width: 15rem;
```

Comment:
This almost equals with `w5` why don't use it?

Reply:
I considered it, but it keeps the container slightly small, a little strange.